chapters/ch13/4.semantic-search.ipynb

```diff
@@ -552,22 +559,23 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "0.723 | How is elevation gain and change measured for hiking trails?\n",
-      "0.715 | How do I Plan a Hiking Trip to Rocky Mountain National Park, CO\n",
-      "0.698 | Hints for hiking the west highland way\n",
-      "0.694 | New Hampshire A.T. Section Hike in May? Logistics and Trail Conditions\n",
-      "0.678 | Long distance hiking trail markings in North America or parts thereof\n",
-      "0.671 | How far is a reasonable distance for someone to hike on their first trip?\n",
-      "0.67 | Vancouver Sunrise Hike Advice?\n",
-      "0.668 | How to prepare on hiking routes?\n",
-      "0.667 | Camping on top of a mountain\n",
-      "0.662 | Advice for first Grand Canyon Hike for Eastern Hikers\n"
+      "0.723 | What should I do if I disturb a bird of prey with a kill?\n",
+      "0.715 | Relation of ski edge sharpening to skiing ability and snow conditions\n",
+      "0.698 | New Suunto Ambit3 and don't know how to move log data from watch to moves count\n",
+      "0.694 | How to get kinks and twists out of a climbing rope?\n",
+      "0.678 | I put a big gouge in the bow of my canoe going across rocks. How can I fix it?\n",
+      "0.671 | Does a tent need special after-trip care when camping on and near ocean beaches?\n",
+      "0.67 | Footwear for one month in wet climate\n",
+      "0.668 | Hiking and wild sleeping in California wilderness\n",
+      "0.667 | Any tips to reduce chance that I have to get up and pee in the night (not counting dehydration, of course)\n",
+      "0.662 | What is the best way to figure out the size/volume of an old pack?\n"
      ]
     }
    ],
    "source": [
     "query = \"mountain hike\"\n",
     "search_results = semantic_search_with_engine(embeddings_collection, query)\n",
+    "print(type(embeddings_collection))\n",
```

Comment:
I assume this was a debugging line (printing out the collection type (Solr, OpenSearch, etc.)) and should be removed.

Reply:
Removed